changelog.md

```diff
@@ -3,7 +3,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-# [Unreleased](https://github.com/MyPureCloud/genesys-cloud-webrtc-sdk/compare/v9.0.5...HEAD)
+# [Unreleased](https://github.com/MyPureCloud/genesys-cloud-webrtc-sdk/compare/v9.0.6...HEAD)
+
+# [v9.0.5](https://github.com/MyPureCloud/genesys-cloud-webrtc-sdk/compare/v9.0.5...v9.0.6)
```

Comment:
Is this labeling right? There is a 9.0.5 underneath but I'm not sure if that was just a mistake from the past

Reply:
🔥

Reply:
> Is this labeling right? There is a 9.0.5 underneath but I'm not sure if that was just a mistake from the past

oops, forgot to update the label.